.circleci/config.yml

```diff
@@ -3,59 +3,70 @@ version: 2
 jobs:
   lint:
     docker:
-      - image: debian:bullseye
+      - image: fedora:32
     steps:
-      - run: apt-get update && apt-get install -y sudo git make
+      - run: dnf install -y make
       - checkout
       - run:
-          name: Install dependencies and run linters
+          name: Install dependencies
           command: |
             make install-deps
-            python3 -m venv .venv
-            source .venv/bin/activate
             pip install --require-hashes -r requirements/dev-requirements.txt
-            sudo apt-get install -y file
-            make black
-  buildrpm:
+      - run:
+          name: Run linters
+          command: make lint
+  build-rpm:
     docker:
-      - image: debian:bullseye
+      - image: fedora:32
     steps:
-      - run: apt-get update && apt-get install -y sudo git make
+      - run: dnf install -y make
       - checkout
-      - run: make install-deps
-      - run: make dom0-rpm
-  reprotest:
+      - run:
+          name: Install dependencies
+          command: |
+            make install-deps
+            pip3 install --require-hashes -r requirements/dev-requirements.txt
+      # In containers that cannot be run with seccomp=unconfined, setarch fails,
+      # breaking reprotest. Symlinking setarch to true makes reprotest believe
+      # everything worked as expected.
+      - run:
+          name: Make reprotest compatible with container use
+          command: sudo ln -s /bin/true /usr/local/bin/setarch
+      - run:
```

Comment:
Unrelated to the CI failure, I would suggest running a plain `make build-rpm` first, to verify that a standalone build works (it's fast enough), then do `make reprotest` for reproducibility. I think that might also fix the current failure that the output path doesn't exist yet, but didn't actually check.